rust/terraform/modules/validator/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "aws_region" {
+  description = "AWS region"
+  type        = string
+}
+
+variable "validator_cluster_id" {
+  description = "ID of the validator cluster"
+  type        = string
+}
+
+variable "validator_subnet_id" {
+  description = "ID of the validator subnet"
+  type        = string
+}
+
+variable "validator_sg_id" {
+  description = "ID of the validator security group"
+  type        = string
+}
+
+variable "validator_nat_gateway_id" {
+  description = "ID of the validator NAT gateway"
+  type        = string
+}
+
+variable "validator_name" {
+  description = "The name of the validator"
+  type        = string
+}
+
+variable "origin_chain_name" {
+  description = "The origin chain of the validator"
+  type        = string
+}
+
+variable "validator_cpu" {
+  description = "CPU units used by the validator. Default 1 vCPU."
+  type        = string
+  default     = "1024"
+}
+
+variable "validator_memory" {
+  description = "Memory units used by the validator. Default 6GB."
```

Comment:
fwiw we can definitely run on less than 6gb but I guess no harm in 6gb, i'm sure it's not very expensive

Reply:
Lemme know what some good numbers are, I used 6gb because it's at like 80% utilisation

Reply:
ah just leave as 6gb in that case